src/postgres/tree/PostgresFunctionTreeItem.ts

```diff
@@ -36,4 +39,12 @@ export class PostgresFunctionTreeItem extends AzureTreeItem<ISubscriptionContext
     public get iconPath(): TreeItemIconPath {
         return getThemeAgnosticIconPath('Collection.svg');
     }
+
+    public async update(newDefinition: string): Promise<string> {
+        this.definition = newDefinition;
+        const client = new Client(this.parent.clientConfig);
+        await client.connect();
+        await client.query(this.definition);
```

Comment:
query updates the definition? Seems odd to me

Reply:
So `definition` is technically the query used to create/replace a function. So executing the definition query will update it on the server.

Would it be helpful to change the `definition` var name?

Reply:
🤷‍♂️